test/utils/template.ts

```diff
@@ -0,0 +1,10 @@
+import path from "path";
+import { SUBMODULE_PREFIX } from "../fixtures/files";
+
+export const toSubmoduleFiles = (files: string[]): string[] => {
+    const submoduleFiles: string[] = [];
+    files.forEach((file) => {
+        submoduleFiles.push(path.join(SUBMODULE_PREFIX, file))
+    })
+    return submoduleFiles;
+};
```

Comment:
Codacy found an issue: [file should end with a newline](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092258)